README.md

```diff
@@ -210,9 +210,11 @@ Separator instances have a property `type` equal to `separator`. This should all
 
 #### List - `{type: 'list'}`
 
-Take `type`, `name`, `message`, `choices`[, `default`, `filter`] properties. (Note that
+Take `type`, `name`, `message`, `choices`[, `default`, `filter`, `loop`] properties. (Note that
 default must be the choice `index` in the array or a choice `value`)
 
+`loop:false` will disable the default looping behavior.
```

Comment:
You should add put this documentation to [Question](https://github.com/SBoudrias/Inquirer.js#question) section.

Reply:
moved